app/views/shared/footer/_second.html.erb

```diff
@@ -39,3 +39,24 @@
     </div>
   </div>
 </div>
+<% if Flipper.enabled?(:download_footer_icons) %>
```

Comment:
not sure we should have this tbh, its not saving much time and won't help with adding other icons to emails, we're going to need to add a set of fuel type and other icons.

Reply:
Was thinking about this overnight. Agree it would be helpful to have the icons easily available. How about this as an alternative:

- update the icon component preview to include a section with all of the icons we need to use in emails, so those for the footer and those you'll need in the alert email changes. The section can have a title and notes about how to download the icons and how that works
- integrate the JS you wrote into that section, only

Gives us a useful developer tool and avoids the need to use a feature flag as its in the preview and not the main application templates. We can easily expand the set of icons displayed to cover all situations.